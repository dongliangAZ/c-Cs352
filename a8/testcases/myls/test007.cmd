vgopts=$1
outfile=$2
errfile=$3
statusfile=$4

ulimit -t 5
thisdir=`pwd`
cd myls-test-dir/SpecialCharFilenames
valgrind $vgopts -q $thisdir/myls ')()()()()('  >${outfile}  2>${errfile}

exitstatus=$?
echo $exitstatus > $statusfile
