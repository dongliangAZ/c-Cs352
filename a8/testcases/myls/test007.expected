#$%^&*@?
$%^&*()_+
.
..
@#$%^^&*
aaa
bbb
ccc
ddd
eee
fff
ggg
hhh
iii
jjj
this filename contains whitespace
